jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -261,7 +261,6 @@ static Unsafe getUnsafe() {
 /*[IF Java11]*/
 	/* Store the results of getNestHostImpl() and getNestMembersImpl() respectively */
```

Comment:
Please remove `getNestMembersImpl()` comment as well.

Reply:
Removed the comment entirely